src/matcha_ml/runners/remote_state_runner.py

```diff
@@ -41,6 +42,11 @@ def _get_terraform_output(self) -> Tuple[str, str, str]:
 
         return account_name, container_name, resource_group_name
 
+    def _clean_up(self) -> None:
+        """Remove the whole .matcha directory when destroy full is run."""
+        matcha_template_dir = os.path.join(os.getcwd(), ".matcha")
+        shutil.rmtree(matcha_template_dir)
```

Comment:
Is the case where this `matcha_template_dir` doesn't exist handled? This will throw a `FileNotFoundError` otherwise